pkg/admission/namespaceconditions/labelcondition.go

```diff
@@ -0,0 +1,120 @@
+package namespaceconditions
+
+import (
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apiserver/pkg/admission"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	corev1lister "k8s.io/client-go/listers/core/v1"
+)
+
+const runLevelLabel = "openshift.io/run-level"
```

Comment:
runlevel?

Reply:
maybe something like `openshift.io/infra-runlevel`

Reply:
runlevel matches upstream docs and gives clear cardinality.  https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/admissionregistration/v1beta1/types.go#L152